test/cb6-but/cb6-but-notcentered.pdb

```diff
@@ -0,0 +1,124 @@
+ATOM      1  C   CB6     1       2.417   4.690   4.881  1.00  0.00           C
```

Comment:
Why do we need this file?

Reply:
Good test case for the align code.  I think the current PDBs in the cb6-but folder were already aligned.